src/bio/bio_xstream.c

```diff
@@ -216,6 +282,20 @@ bio_spdk_env_init(void)
 		return rc;
 	}
 
+	if (spdk_conf_find_section(NULL, "Vmd") != NULL) {
+		/**
+		 * Enumerate VMD devices and hook them into the SPDK PCI
+		 * subsystem.
+		 */
+		rc = spdk_vmd_init();
```

Comment:
Did you try not call this spdk_vmd_init() explicitly?

Reply:
Yes, and it fails and SPDK is unable to use the VMD devices. Seems like it is a necessary call in addition to the spdk_bdev_init